src/App.tsx

```diff
@@ -1,47 +1,110 @@
-import React, { FC } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 import './App.css';
-import Header from './components/Header';
+import axios from 'axios';
+import queryString from 'query-string';
+import { Innholdsformasjon } from './types';
 import CollapsedBox from './components/CollapsedBox';
-import { TitleLabel, AuthorLabel, ISBNLabel, ImageContainer } from './components/CustomElements';
-import pelsjegere from './resources/pelsjegere.jpg';
+import Header from './components/Header';
+import { AuthorLabel, ErrorTextField, ImageContainer, ISBNLabel, TitleLabel } from './components/CustomElements';
 
 const URL = window.location.href;
+const apiUrl = process.env.REACT_APP_INNHOLDSTJENESTE_API_URL;
+const imageUrl = process.env.REACT_APP_INNHOLDSTJENESTE_IMAGES_URL;
+const queryParams = queryString.parse(window.location.search);
```

Comment:
new URLSearchParams(location.search) er innebygd funksjonalitet som gjør alt som "query-string" npm-pakken gjør, og som er designet for å ta over "query-string"

Reply:
Takk for gode innspill, Anette! :) Vil du ta en liten session og vise meg og forklare hvordan jeg skal ta inn de to forslagene?

Reply:
Det kan jeg